boards/arm/xenvm/xenvm.dts

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2020 EPAM Systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * This file was created by running
+ * # LIBXL_DEBUG_DUMP_DTB=domu-libxl.dtb xl create zephyr.conf
+ * decompilling resulting domu-libxl.dtb and then munally aligning it
+ * with zephyr requirements.
```

Comment:
Does it mean Zephyr is not able to detect the amount of memory at runtime?

Reply:
Yes. AFAIK, Zephyr uses static configuration.